nntrainer/tensor/blas_interface.h

```diff
@@ -35,77 +36,93 @@ enum CBLAS_TRANSPOSE {
 #include <helper_functions.h>
 #endif
 
+#include <cstdint>
 #include <tensor_dim.h>
-
 namespace nntrainer {
 
 #ifdef ENABLE_FP16
+/** @brief sscal computation */
```

Comment:
It might be better to write the brief more formally.
For example,
/**
 * @brief     sgemm computation with neon : Y = alpha*op(A)*op(B) + beta*C,
 * where op(X) is one of X or X**T
 * @param[in] A __fp16 * for Matrix A
 * @param[in] B __fp16 * for Matrix B
 * @param[in] C __fp16 * for Matrix C
 * @param[in] M number of op(A)'s and C's row
 * @param[in] N number of op(B)'s and C's columns
 * @param[in] K number of op(A)'s and columns and op(B)'s rows
 * @param[in] alpha float number
 * @param[in] beta float number
 */

or 
you can use @copydoc as well.

Reply:
We have not been using @brief in blas_interface, but recent CI test is asking me to do so for some reaseon.
I temporally added a skeleton breif, but wondering if this is a good behavior or not.
Is it recommended to proceed with it?

Reply:
We have to add doxygen comment in every function definition.